server/src/test/java/io/druid/client/CachingClusteredClientTest.java

```diff
@@ -205,7 +205,7 @@
       )
   );
   private static final List<AggregatorFactory> RENAMED_AGGS = Arrays.asList(
-      new CountAggregatorFactory("rows2"),
+      new CountAggregatorFactory("rows"),
```

Comment:
Ah I see what's going on. There's nothing wrong with verbose methods (usually I prefer them) so that's fine.